### ISSUE TITLE
Improve Screen Reader Accessibility

### ISSUE THREAD

**Issue by C-Loftus** [talonvoice/talon#641]
Currently, the built in log viewer is not accessible for those with a vision impairment.  As far as I can see on my computer running Windows with NVDA, nothing can be read out loud. The title of the window is also not updated.  Given the fact that reading from the log internally within Talon can cause an infinite loop,  the user has to use external add-ons for their screen reader to read the log and this may not be a practical solution given the burden of installing user add ons and developing them for different screen readers. (JAWS Orca NVDA VoiceOver etc) 

I am not sure if it is feasible cross platform, but if possible it would be helpful to have the title of the log viewer window updated based on the last log output. ( So in the case of an error it would obviously not have the entire stack trace but just the last line or something similar) 

There are also probably other ways to make it more directly accessible within the viewer itself but I am not sure how difficult that would be to change.

**Comment by C-Loftus** [comment]
Following up on this, it appears as well the windows from imgui are not accessible with screen readers either.

**Comment by C-Loftus** [comment]
The update window is also not accessible which may be confusing since it can autostart on talon launch
